network-api/networkapi/wagtailpages/models.py

```diff
@@ -618,43 +620,29 @@ def filter_entries_for_tag(self, entries, context):
         return entries
 
     def filter_entries_for_category(self, entries, context):
-        """
-        NOTE: we currently assume only blog pages can have
-        categories. If that ever changes, we will need to rename
-        the model from BlogPageCategory to PageCategory and make
-        the corresponding adjustments to this code
-        """
-        slug = self.filtered.get('category')
-        cat = None
-        for bpc in BlogPageCategory.objects.all():
-            # We can't use .filter for @property fields,
-            # so we have to run through all categories =(
-            if bpc.slug == slug:
-                cat = bpc
-
-        if cat is not None:
-            # make sure we bypass "x results for Y"
-            context['no_filter_ui'] = True
-
-            # and that we don't show the primary tag/category
-            context['hide_classifiers'] = True
-
-            # explicitly set the index page title and intro
-            print('titlecase')
-            context['index_title'] = titlecase(f'{cat.name} {self.title}')
-            context['index_intro'] = cat.intro
-
-            # and then the filtered content
-            context['terms'] = [cat.name, ]
-            entries = [
-                entry
-                for
-                entry in entries.specific()
-                if
-                hasattr(entry, 'category')
-                and
-                cat in entry.category.all()
-            ]
+        category = self.filtered.get('category')
```

Comment:
we can't get here anymore is the category doesn't exist, so the "if cat is not None" scoping became irrelevant